piptools/repositories/pypi.py

```diff
@@ -257,10 +260,18 @@ def get_hashes(self, ireq):
             ireq.specifier.filter((candidate.version for candidate in all_candidates)))
         matching_candidates = candidates_by_version[matching_versions[0]]
 
-        return {
-            self._get_file_hash(candidate.location)
-            for candidate in matching_candidates
-        }
+        log.debug('  {}'.format(ireq.name))
+
+        if log.verbosity >= 1:
+            context_manager = progressbar(matching_candidates)
```

Comment:
Not covered?

Reply:
Huh, indeed it is. I'll add a test then. Thank you for good catch.

Reply:
At some point i was sure it was covered. But it's obvious by the falling checks :)